charts/zalenium/templates/_pod-template.yaml

```diff
@@ -59,7 +59,7 @@ spec:
         timeoutSeconds: {{ .Values.hub.livenessTimeout }}
       readinessProbe:
         httpGet:
-          path: {{ if and (.Values.ingress.path) (not (eq .Values.ingress.path "/")) }}{{ .Values.ingress.path }}{{ end }}/grid/console
+          path: {{ if (.Values.ingress.path) and (not (eq .Values.ingress.path "/")) }}{{ .Values.ingress.path }}{{ end }}/grid/console
```

Comment:
Same comment

Reply:
Yes I did test this.  if `and` is in the front and .Values.ingress.path not defined, it wasn't working for me.  See previous comment for error

Reply:
Same simplfication